core/broadleaf-framework-web/src/main/java/org/broadleafcommerce/core/web/controller/account/AbstractCustomerAddressController.java

```diff
@@ -87,8 +91,8 @@ protected void initBinder(HttpServletRequest request, ServletRequestDataBinder b
         initBinderService.configAddressInitBinder(binder);
     }
 
-    protected List<State> populateStates() {
-        return stateService.findStates();
+    protected List<CountrySubdivision> populateStates() {
+        return countrySubdivisionService.findSubdivisions();
```

Comment:
Lets also revert this change. It looks like that in our demo we extend this and set a model attribute on it. Also CountrySubdivision != State therefore the front end mappings setup in demo site aren't setup correctly for this change (even in the PR). We'll catch this with the next pass when we do 6.2